source/agora/node/Ledger.d

```diff
@@ -1572,24 +1572,23 @@ unittest
     import agora.consensus.data.genesis.Coinnet : CoinGenesis = GenesisBlock;
```

Comment:
Can't we remove `CoinGenesis`, using `GenesisBlock` instead?

Reply:
No because it is still used (we need two Genesis block to have a mismatch).